src/createGridComponent.js

```diff
@@ -272,22 +272,28 @@ export default function createGridComponent({
         estimatedTotalHeight > height ? scrollbarSize : 0;
 
       this.scrollTo({
-        scrollLeft: getOffsetForColumnAndAlignment(
-          this.props,
-          columnIndex,
-          align,
-          scrollLeft,
-          this._instanceProps,
-          verticalScrollbarSize
-        ),
-        scrollTop: getOffsetForRowAndAlignment(
-          this.props,
-          rowIndex,
-          align,
-          scrollTop,
-          this._instanceProps,
-          horizontalScrollbarSize
-        ),
+        scrollLeft:
+          columnIndex !== undefined && columnIndex >= 0
```

Comment:
Why is the `>= 0` needed?

Reply:
Well, it's not needed for this feature, but negative indexes will not produce valid results, especially with `VariableSizeGrid`. I can remove the check if you prefer to leave the behaviour undefined.

Reply:
@bvaughn Specifically, for `VariableSizeGrid`, [`itemMetadata`](https://github.com/bvaughn/react-window/blob/master/src/VariableSizeGrid.js#L240) will be `undefined`, which will throw an error [when calculating `maxOffset`](https://github.com/bvaughn/react-window/blob/master/src/VariableSizeGrid.js#L251).